tests/kda/test_kda_output_only.py

```
@@ -0,0 +1,355 @@
+        torch.randn(B, T, HV, 128, dtype=torch.float32, device=q.device)
+    ).to(torch.bfloat16)
+    buf = torch.empty(B, T, HV, 128, dtype=torch.bfloat16, device=q.device)
+    out = kda_output_only_decode(q, k, v, g_log, beta, h0, idx, output=buf)
+    assert out.data_ptr() == buf.data_ptr()
+    ref = kda_output_only_decode(q, k, v, g_log, beta, h0, idx)
+    torch.testing.assert_close(out.float(), ref.float(), atol=0.0, rtol=0.0)
+
+
+def ref_corrections(q, k, v, g_log, beta, h0, idx, scale):
+    """fp32 reference also returning per-token corrections U_t = beta*(v-u)."""
+    B, T, H, K = q.shape
```

Ruff (0.16.2)

[warning] Unpacked variable `K` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)